packages/lesswrong/components/common/TabNavigationMenu/SubforumsList.tsx

```diff
@@ -14,24 +14,36 @@ const styles = ((theme: ThemeType): JssStyles => ({
     paddingRight: 0,
     '&:hover': {
       backgroundColor: 'transparent' // Prevent MUI default behavior of rendering solid background on hover
-    }
+    },
+    whiteSpace: 'break-spaces',
+    height: 'unset',
+    minHeight: 24,
+  },
+  title: {
+    paddingLeft: 62,
+    paddingBottom: 5,
+    ...theme.typography.body2,
+    color: theme.palette.grey[800],
   },
   subItem: {
     textTransform: 'capitalize',
-    textOverflow: "ellipsis",
+    whiteSpace: 'break-spaces !important',
```

Comment:
Allowing long names to wrap around so that the unread count still shows up at the end